src/components/Cart/CartDialogs.js

```diff
@@ -0,0 +1,114 @@
+import { React } from 'react';
+import {
+  Typography, Dialog, DialogContent, DialogActions, Button, makeStyles,
+} from '@material-ui/core';
+import PropTypes from 'prop-types';
+import './Cart.css';
+
+const useStyles = makeStyles({
+  alertCard: {
+    maxWidth: '350px',
```

Comment:
I think dialogs default to a certain percentage of the screen size, but I wanted it to be small even on large screens, so the size of the dialog will adjust on smaller screens, but it will not be larger than 350px on large ones.

Reply:
I see what you mean, and you make a great point. In this case, we can keep this and make note that all the minWidth/min properties can be set to a constant px value if needed